wp-modules/editor/tests/ModelTest.php

```diff
@@ -16,6 +20,34 @@
  */
 class ModelTest extends WP_UnitTestCase {
 
+	/**
+	 * @inheritDoc
+	 */
+	public function setUp(): void {
+		parent::setUp();
+		$this->stylesheet_dir = get_wp_filesystem_api()->wp_themes_dir() . '/pm-testing';
+		get_wp_filesystem_api()->mkdir( $this->stylesheet_dir );
+		add_filter( 'stylesheet_directory', [ $this, 'get_stylesheet_dir' ] );
```

Comment:
Like [another test](https://github.com/studiopress/pattern-manager/blob/21641406d649b9f66473e9cf9e2e5df44612acad/wp-modules/pattern-data-handlers/tests/PatternDataHandlersTest.php#L350), this makes [get_patterns_directory()](https://github.com/studiopress/pattern-manager/blob/21641406d649b9f66473e9cf9e2e5df44612acad/wp-modules/pattern-data-handlers/pattern-data-handlers.php#L193) get a mock theme we created for these unit tests.